install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
 $ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/byo/openshift-cluster/openshift-metrics.yml \
    -e openshift_metrics_install_metrics=False
 ----
+
+[[openshift-prometheus]]
+== Prometheus on OpenShift
+
+Prometheus is a stand-alone, open source systems monitoring and alerting
+toolkit. You can use Prometheus to visualize metrics and alerts for OpenShift
+system resources.
+
+[IMPORTANT]
+====
+The Prometheus is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+[openshift-prometheus-roles]
+=== Setting Prometheus Role Variables
+
+The Prometheus role creates:
+
+* OpenShift-Prometheus namespace.
+* Prometheus clusterrolebinding and service account.
+* PVs for Prometheus, alertmanager and alertbuffer for internal NFS.
+* Prometheus pod with Prometheus behind oauth-proxy, alertmanager and alert-buffer behind oauth-proxy.
+* Prometheus and alertmanager configmaps.
+* Prometheus and alerts services and direct routes.
+* Prometheus, alertmanager and alert-buffer PVCs.
+
+Depends on `openshift_hosted_prometheus_deploy` flag.
+
+Set the following role variables to install and configure Prometheus.
+
+.Prometheus Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_prometheus_state`
+|Set variable to install and update. Remove variable to uninstall.
+
+|`openshift_prometheus_namespace`
+|Project namespace where the components will be deployed. Default set to
```

Comment:
default value for openshift_prometheus_namespace is openshift-metrics, just changed from Nov. 10th, 2017

Reply:
Is this correct now? With: `openshift_prometheus_namespace=${USER_PROJECT}`